content/md/user-management/troubleshoot-the-sso.md

```diff
@@ -0,0 +1,58 @@
+---
+id: troubleshoot-the-sso
+themes: user-management
+title: How to **troubleshoot** the Single Sign-On
+popular: false
+ee-only: true
+related: configure-saml-sso
+priority: low
+---
+
+# Overview
+
+Single Sign-On is a powerful tool for productivity but sometimes, as multiple applications are concerned, it can be a hassle to troubleshoot when something goes wrong with it.
```

Comment:
you could simply say (at the end): "when something goes wrong." and remove "with it"

Reply:
fixed